### PR TITLE
added more detailed instruction for binary installation

### LINKED ISSUE #936: Expand binary installation instructions
Explain steps for installing the binary with or without installation to GOPATH.

### PR DESCRIPTION
**Pull request by ijonele**
## Summary by CodeRabbit

- **Documentation**
	- Improved clarity and accuracy in the instructions for creating and managing a Keychain.
	- Enhanced guidance for joining the Buenavista testnet, including a new "Overview" section and updated installation steps.
	- Added tips and detailed instructions for binary installation and configuration settings.
	- Clarified command sequences and emphasized expected output formats for better user understanding.
	- Expanded formatting for improved readability and usability of the documentation.

### CODE REVIEW

File: docs/developer-docs/docs/build-a-keychain/operate-a-keychain/create-a-keychain.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ import TabItem from '@theme/TabItem';
 
 To become a **Keychain operator**, you need to create and configure a Keychain entity on-chain, as shown in this guide. To interact with the chain,  you'll use [node commands](/operate-a-node/node-commands).
 
-You can either run a [local chain](/operate-a-node/run-a-local-chain) to test your configuration or interact with [Buenavista testent](http://localhost:3000/operate-a-node/buenavista-testnet/join-buenavista). In the provided code snippets, you'll find tabs with different versions of node commands.
+You can either run a [local chain](/operate-a-node/run-a-local-chain) to test your configuration or interact with [Buenavista testent](/operate-a-node/buenavista-testnet/join-buenavista). In the provided code snippets, you'll find tabs with different versions of node commands.
```
Comment (jlehtimaki) [inline docs/developer-docs/docs/build-a-keychain/operate-a-keychain/create-a-keychain.md]: Typo in: Testent

File: docs/developer-docs/docs/build-a-keychain/operate-a-keychain/create-a-keychain.md [hunk 2]
```diff
@@ -68,13 +68,14 @@ You can skip this guide and test a preconfigured Keychain. Just run a local node
 
 1. If you haven't yet, [install Go](https://golang.org/doc/install) 1.22.3 or later and [just](https://github.com/casey/just) 1.34.0 or later.
 
-2. Clone the repository with Warden source code and initialize the chain home folder:
+2. Clone the repository with Warden source code. Then build the binary and initialize the chain home folder:
```
Comment (jlehtimaki) [inline docs/developer-docs/docs/build-a-keychain/operate-a-keychain/create-a-keychain.md:71]: Initialize the network, not the home folder

File: docs/developer-docs/docs/operate-a-node/buenavista-testnet/join-buenavista.md [hunk 3]
```diff
@@ -41,22 +41,46 @@ To be able to interact with the node, install `wardend` (the Warden binary) usin
 
 1. Download the binary for your platform from the [release page](https://github.com/warden-protocol/wardenprotocol/releases/tag/v0.4.2) and unzip it. The archive contains the `wardend` binary.
 
-2. Initialize the chain home folder:
+2. Navigate to the directory containing the binary and initialize the chain home directory:
```
Comment (jlehtimaki) [inline docs/developer-docs/docs/operate-a-node/buenavista-testnet/join-buenavista.md]: We initialize the node not the home folder

File: docs/developer-docs/docs/operate-a-node/buenavista-testnet/join-buenavista.md [hunk 4]
```diff
@@ -41,22 +41,46 @@ To be able to interact with the node, install `wardend` (the Warden binary) usin
...
 ### Option 2: Use the source code
 
-Build the `wardend` binary and initialize the chain home folder:
+1. Clone the repository and navigate to the root directory:
 
-```bash
-git clone --depth 1 --branch v0.4.2 https://github.com/warden-protocol/wardenprotocol
-just build
+   ```bash
+   git clone --depth 1 --branch v0.4.2 https://github.com/warden-protocol/wardenprotocol
```
Comment (jlehtimaki) [inline docs/developer-docs/docs/operate-a-node/buenavista-testnet/join-buenavista.md:60]: Wrong version?

File: docs/developer-docs/docs/operate-a-node/buenavista-testnet/join-buenavista.md (hunk 3 above)
Comment (ijonele) [reply docs/developer-docs/docs/operate-a-node/buenavista-testnet/join-buenavista.md]: Yes, it looked strange to me too, but I wasn't sure. I'll correct it

File: docs/developer-docs/docs/operate-a-node/buenavista-testnet/join-buenavista.md (hunk 4 above)
Comment (ijonele) [reply docs/developer-docs/docs/operate-a-node/buenavista-testnet/join-buenavista.md:60]: It's the current buenavista guide, so I think it's correct. I just split this set of commands into two parts. In the Chiado article, it'll be the v0.5.1 branch

Comment (jlehtimaki) [reply docs/developer-docs/docs/operate-a-node/buenavista-testnet/join-buenavista.md:60]: Yes it is correct

**Review by jlehtimaki — APPROVED** [review]

**Pull request merged**